testUtils/api.ts

```diff
@@ -1,30 +1,44 @@
 import { ApolloClient, InMemoryCache } from "@apollo/client";
 import { BatchHttpLink } from "@apollo/client/link/batch-http";
 import { getApiUrl } from "@dashboard/config";
-import NodeHttpAdapter from "@pollyjs/adapter-node-http";
-import FSPersister from "@pollyjs/persister-fs";
 import { createFetch } from "@saleor/sdk";
+import { isCI } from "ci-info";
+import { get } from "env-var";
 import path from "path";
 import { setupPolly } from "setup-polly-jest";
 
+const POLLY_MODES = ["replay", "record", "passthrough", "stopped"] as const;
+
+const POLLY_MODE = get("POLLY_MODE")
```

Comment:
**issue**: do you need a library for getting env value? we access process.env actually, node.js runs this, if we introduce a new library we also have to plan it to apply everywhere to have one way of accessing vars

Reply:
The main idea here for this library was to abstract away the tricky part which is parsing of those values:
- we want `"false"` and `"true"` to be mapped to their correct boolean types
- we want to check if `POLLY_MODE` has the correct value declared in the union (otherwise raise exception)
- we want to provide a default value if none is provided by env var

Of course, we could do that by accessing raw env vars, like [what we have in saleor-app-checout](https://github.com/saleor/react-storefront/blob/main/apps/saleor-app-checkout/test-utils.ts#L181). But comparing that to the implementation with `env-var` it is much easier to understand and less error-prone in my opinion (we had to fix the implementation in checkout a couple of times 🙈).

We don't need to use that everywhere we use an env var, just where you need to parse it in some way.